nearcache/nearcache_it_test.go

```diff
@@ -1097,6 +1097,23 @@ func TestRepairingTaskRun(t *testing.T) {
 	time.Sleep(2 * time.Second)
 }
 
+func TestSameNearCacheBackingStoreForEachGetMap(t *testing.T) {
```

Comment:
I ran this test on current master it passes. What do we fix in this PR?

Reply:
I've changed the test to check for a memory leak at: https://github.com/hazelcast/hazelcast-go-client/pull/921/commits/5fd475e1bc74e079661db9365f3554c6f181862f